toolbox-app/package.json

```
@@ -1,4 +1,6 @@
-{
+{ 
+  "homepage": "https://doloresax.github.io/ROM_ToolsBox/",
```

Biome (1.9.4)

[error] The key homepage was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)